drivers/spi/spi_ll_stm32.c

```diff
@@ -28,10 +28,23 @@ LOG_MODULE_REGISTER(spi_ll_stm32);
 
 #ifdef CONFIG_NOCACHE_MEMORY
 #include <zephyr/linker/linker-defs.h>
+#elif defined(CONFIG_CACHE_MANAGEMENT)
+#include <zephyr/arch/cache.h>
 #endif /* CONFIG_NOCACHE_MEMORY */
 
 #include "spi_ll_stm32.h"
 
+/* Check defined(CONFIG_DCACHE) because some platforms disable it in the tests
+ * e.g. nucleo_f746zg
```

Comment:
It could be interesting to find out why and fix it if it has no sense anymore.

Reply:
I'm afraid I don't have access to f746zg, and so I'm unable to test this.